rep-0149.rst

```diff
@@ -51,6 +51,12 @@ Motivation
 This REP covers three separate topics which are described in the following
 subsections.
 
+A source tree containing a conformant ``package.xml`` file in its root
```

Comment:
I think this is likely incorrect. I think Dirk will come over and mention that the package.xml has nothing to do with ROS, it has to do with catkin / ament.

Reply:
I think the term "ROS Package" has to be this broad for the following statement to be correct:

> **<build_depend> (multiple)**

> Declares a rosdep key or ROS package name that this package requires at build-time. For system packages, the rosdep key will normally specify the "development" package, which frequently ends in "-dev".

This REP does use the term "catkin package", which seems to mean "a ROS package that uses  `build_type` = `catkin`"

Reply:
This clarifications you added seem to be to be largely unnecessary. Just one guys' opinion. It might help if you added some additional context about "why" this is necessary.

Reply:
I think "ROS package" is unambiguous. After all this is a "ROS Enhancement Proposal". It might be helpful to think of the "ROS" qualifier as "interoperable with the ROS community and associated tools".

I don't think it's sensible to just drop the qualifier. "Package" a very overloaded term, and these things are often contained *within* ROS packages:

* Python: [**distribution package** or an **import package**](https://packaging.python.org/glossary/#term-import-package)
* CMake: [**Config-file Package or a Find-module Package**](https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#using-packages)
* Anything installed by a package manager, including things to satisfy system dependencies.

For instance `rosidl_generator_c`:

* Is contained within a **ROS package** named `rosidl_generator_c`
* Is built using a **CMake project** named `rosidl_generator_c`
* Installs a **CMake config-file package** named `rosidl_generator_c`
* Installs a **Python import package** named `rosidl_generator_c`